integration_test/third_party_apps_test.go

```diff
@@ -658,6 +658,17 @@ func modifiedFiles(t *testing.T) []string {
 // Checks the extracted app names against the set of all known apps.
 func determineImpactedApps(mf []string, allApps map[string]metadata.IntegrationMetadata) map[string]bool {
 	impactedApps := make(map[string]bool)
+	for _, f := range mf {
+		// File names: submodules/fluent-bit
+		if strings.HasPrefix(f, "submodules/") {
+			for app, _ := range allApps {
+				impactedApps[app] = true
+			}
+			log.Printf("impacted apps: %v", impactedApps)
```

Comment:
I would do this as:
```
defer log.Printf("impacted apps: %v", impactedApps)
```
right after declaring impactedApps. Then you don't have to repeat this here and down below.

Reply:
Done!